addons/web/static/src/core/l10n/numbers.js

```diff
@@ -4,6 +4,8 @@ import { escapeRegExp, intersperse, sprintf } from "../utils/strings";
 import { localization } from "./localization";
 import { _lt } from "./translation";
 
+export class ParsingError extends Error {}
```

Comment:
I think the error should not refer to the concept of parsing, but to the concept of invalid number. Why not `InvalidNumberError`